### ISSUE TITLE
Message Bar Text Is Not Focusable

### ISSUE THREAD

**Issue by tabrumle** [microsoft/fluentui#21757]
### Environment Information

- **Package version(s)**: V8
- **Browser and OS versions**: All

### Describe the issue:

Message bars do not have a way to give narrator access to their underlying text if there is no executable item within them. The best we can do is make an alert on render, but this still leaves a gap where users who rely on screen readers may not be aware of a constant message bar that has important information in it.

### Please provide a reproduction of the issue in a codepen:

[Repro](https://codepen.io/The_Golden_Goose/pen/gOXGqjM)

#### Actual behavior:

Message Bar text cannot be focused.

#### Expected behavior:

There is a way to narrator navigate to the text within a message bar after it is rendered.

**Comment by Rafael805** [comment]
Are there any updates here?

**Comment by tabrumle** [comment]
@Rafael805 I was able to get around this by adding a tabIndex of 0 on the messagebar. This isn't the exact behavior you may want but should be a temp solution. @tomi-msft is this still something you are planning on getting to?

**Comment by rhernandez-msft** [comment]
@tabrumle interesting idea.

Are you doing the following:

_Example:_
`
<MessageBar messageBarType={MessageBarType.success} tabIndex={0}>   
    Some text
</MessageBar>
`

I tried something similar, and it wasn't working for me unless I missed something.
